### PR TITLE
OADP 1178 - featureFlags

### LINKED ISSUE #54575: [enterprise-4.11] Issue in file backup_and_restore/application_backup_and_restore/installing/installing-oadp-ocs.adoc
### Which section(s) is the issue in?
"Enabling CSI in the DataProtectionApplication CR" section
Also the other pages "Installing and configuring OADP with ***" are affected.

### What needs fixing?
The featureFlags field is placed wrong in documentation. `spec.configuration.featureFlags` does not exist.
Correct is:
```
spec:
  configuration:
    velero:
      featureFlags:
      - EnableCSI 
```

### PR DESCRIPTION
**Pull request by CarmiWisemon**
OADP - 1.2.0

OCP - 4.9+

Resolves - https://issues.redhat.com/browse/OADP-1178

Deploy preview - https://57763--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-enabling-csi-dpa_installing-oadp-aws

### CODE REVIEW

**Review by kaovilai — APPROVED** [review]

**Review by PrasadJoshi12 — APPROVED** [review]

**Comment by sbeskin-redhat** [comment]
@CarmiWisemon 
Looks fine.
Just change the preview link in the PR description to this:
https://57763--docspreview.netlify.app/openshift-enterprise/latest/backup_and_restore/application_backup_and_restore/installing/installing-oadp-aws.html#oadp-enabling-csi-dpa_installing-oadp-aws

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,4 @@ bin
 commercial_package
 .vscode
 .vale
+.idea/
```
Comment (mburke5678) [inline .gitignore:19]: @CarmiWisemon Does this change need to be in this PR?

Comment (sbeskin-redhat) [reply .gitignore:19]: @CarmiWisemon 
Actually, you don't have to add .idea to the gitignore file. It's already there in the openshift-docs repository

Comment (CarmiWisemon) [reply .gitignore:19]: @sbeskin-redhat It adds it by itself.

Comment (CarmiWisemon) [reply .gitignore]: @mburke5678 It has been removed and you can ignore it.

Comment (mburke5678) [reply .gitignore]: @CarmiWisemon Can we get the .gitignore out of the PR?

Comment (CarmiWisemon) [reply .gitignore]: @shahsahil264 @mburke5678 Can we just delete it here?

Comment (CarmiWisemon) [reply .gitignore]: @mburke5678 I deleted the file from the PR

Comment (kaovilai) [reply .gitignore]: I don't think that's what they meant.. deleting the file. They just meant you don't have to make the change. aka keep the file as it was.

**Comment by kaovilai** [comment]
Here, merge this.
https://github.com/CarmiWisemon/openshift-docs/pull/1
And we should be good on gitignore stuff.

**Comment by CarmiWisemon** [comment]
> Here, merge this. [CarmiWisemon#1](https://github.com/CarmiWisemon/openshift-docs/pull/1) And we should be good on gitignore stuff.

@mburke5678 can you review to merge the PR Tiger created?

**Pull request closed without merging**